Automattic/woocommerce-payments
# Add Payment Activity's empty state and filters [WIP]

Fixes #8198 

#### Changes proposed in this Pull Request

- Introducing an empty state based on TPV for account

#### Testing instructions





- Get server PR 4775
- Open Payments > Overview
- You should see a new card for Payments Activity with "You have some money". This will be replaced with actual graphs
- You should see an empty state if you don't have a TPV in your account. If you want to force it, change `tpv === 0` to `tpv !== 0`

-------------------


**Post merge**
